spec/spec_helper.rb

```diff
@@ -1,3 +1,5 @@
+require 'factory_girl_rails'
+require 'rails_helper'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.